code/modules/organs/internal/liver.dm

```diff
@@ -39,20 +39,20 @@
 	// Robotic organs filter better but don't get benefits from dylovene for filtering.
 	if(BP_IS_ROBOTIC(src))
 		filter_effect += 1
-	if(getToxLoss() >= 60) //Too many toxins to process. Abort, abort.
```

Comment:
120 is a number we already tried. It doesn't work because that's 2 or more fucked up organs and that's an extremely unreasonable amount of toxloss to get.